build.gradle

```diff
@@ -4,46 +4,40 @@ buildscript {
             url "http://dl.bintray.com/jmoore/java-lib"
         }
     }
+    //noinspection GroovyAssignabilityCheck
     dependencies {
-        classpath 'com.mooregreatsoftware:gradle-defaults:1.0.3'
+        classpath 'com.mooregreatsoftware:gradle-defaults:1.0.6'
     }
 }
 
 plugins {
     id 'com.jfrog.bintray' version '1.4'
+    id "us.kirchmeier.capsule" version "1.0.2"
 }
 
 group = 'com.twcable.grabbit'
 description = 'Command line interface for interacting with the Grabbit Client'
 
 apply plugin: 'groovy'
 
-apply plugin: 'application'
-apply plugin: 'groovy'
-mainClassName = "com.twcable.grabbit.cli.Grabbit"
-
+//noinspection GroovyAssignabilityCheck
 dependencies {
-    compile "org.codehaus.groovy:groovy-all:2.4.3"
+    compile "org.codehaus.groovy:groovy-all:2.4.6"
     compile 'commons-cli:commons-cli:1.2'
 
     // @Nonnull @Nullable etc annotations
-    compile "com.google.code.findbugs:jsr305:2.0.0"
+    // compile "com.google.code.findbugs:jsr305:2.0.0"
```

Comment:
!fixed It got removed later as part of the `to-java` branch (in favor of the Checker Framework), but I've removed in this branch as well.